tests/CoreFX/CoreFX.issues.json

```diff
@@ -59,10 +59,67 @@
                 {
                     "name": "System.Runtime.Serialization.Formatters.Tests.BinaryFormatterTests.ValidateAgainstBlobs",
                     "reason": "The stored blob for type System.CodeDom.MemberAttributes is outdated and needs to be updated."
+                },
+                {
+                    "name": "System.Runtime.Serialization.Formatters.Tests.BinaryFormatterTests.ValidateEqualityComparersAgainstBlobs",
+                    "reason": "outdated"
+                },
+                {
+                    "name": "System.Runtime.Serialization.Formatters.Tests.BinaryFormatterTests.ValidateBasicObjectsRoundtrip",
+                    "reason": "outdated"
+                },
+                {
+                    "name": "System.Runtime.Serialization.Formatters.Tests.BinaryFormatterTests.RoundtripManyObjectsInOneStream",
+                    "reason": "outdated"
```

Comment:
Why are these "outdated"?  What changed?  They're still enabled in corefx, so is this a case of our re-enabling them once coreclr updates its copies of tests?

Reply:
These tests call implementation details and depend on internals visible attribute to do so. The strong names used to build the tests changed and the access to internals fails now.

It was caused by https://github.com/dotnet/corefx/pull/33145 or one of the related change.

It works in corefx because both the product change and test change was done at the same time.

Reply:
> is this a case of our re-enabling them once coreclr updates its copies of tests?

Right.

Reply:
> Right

Ok.  It might be helpful moving forward if we have some kind of convention around the disable reasons.  It wasn't clear to me whether "outdated" meant "this is temporarily invalid until we pull in updated tests" vs "this test has been deleted in corefx and is no longer valid".

Reply:
"outdated" means "this is temporarily invalid until we pull in updated tests drop".

The root cause is always CoreFX product change that is incompatible with the test. I do not think it is interesting to track here how the test was fixed in CoreFX (deleted, updated, renamed, build script changed, ... it is often a combination). 

Also, this is point in time problem. We plan to pick up up-to-date CoreFX test drops together with CoreFX product drops to addresses this issue in more sustainable way. We were very close to turn it on, but it was derailed by Arcade migration. Once the Arcade dust settles down, we will go back to it.

Reply:
> I do not think it is interesting to track here how the test was fixed in CoreFX (deleted, updated, renamed, build script changed, ... it is often a combination).

I'm just thinking about how this file gets updated when we pull in the updated tests, and how the person doing so figures out which can be removed from the file, which can be re-enabled, and which should be left disabled.  If the answer is always "we just wipe this file clean because the root cause is always an incompatibility between the the corefx src and stale tests", then I agree, the reason here doesn't matter.

> We plan to pick up up-to-date CoreFX test drops together with CoreFX product drops to addresses this issue in more sustainable way

Great.

Reply:
> we just wipe this file clean 

Yes, it should be the plan.

Reply:
Great.